CMakeLists.txt

```
@@ -18,7 +18,7 @@
 # limitations under the License.
 cmake_minimum_required(VERSION 3.12)
 project(foundationdb
-  VERSION 6.1.0
+  VERSION 6.2.2
```

```suggestion
  VERSION 6.2.0
```

Let us do this as 6.2.0 for now, as it's going to be inaccurate with what is tagged regardless of how we set it, and I'll try to get our release documentation updated to have this bumped forwards along with versions.target